apps/web/src/components/Header/HeaderActions.tsx

```diff
@@ -0,0 +1,9 @@
+import { Flex } from "@chakra-ui/react";
+
+import { ColorSchemeModeToggle } from "../ColorSchemeModeToggle/ColorSchemeModeToggle";
+
+export const HeaderActions = () => (
```

Comment:
Since it's already stored in the Header directory, maybe just `Actions`?
Also, a separate component doesn't look quite helpful either and can be inlined, WDYT?

Reply:
I think is better to move actions logic to separate component because it can get complicated

Reply:
and besides, I think it's easier to find the right component when searching for a component in the codebase